org.flatpak.Builder.yml

```
@@ -131,7 +131,8 @@ modules:
         x-checker-data:
           type: json
           url: https://api.github.com/repos/flatpak/flatpak/releases
-          version-query: sort_by(.tag_name | sub("^v"; "") | split(".") | map(tonumber)) | last | .tag_name
+          version-query: sort_by(.tag_name | sub("^v"; "") | split(".") | map(tonumber))
+            | last | .tag_name
```

To prevent yaml writer from breaking the line, we can do
```suggestion
          version-query: >-
            sort_by(.tag_name | sub("^v"; "") | split(".") | map(tonumber)) | last | .tag_name
```

---

That makes sense. Maybe even manual line breaks at pipes